### PR TITLE
chore(license): add a new ESLint rule to check license header

### PR DESCRIPTION
**Pull request by skamril**

### CODE REVIEW

**Review by MartinBelthle — DISMISSED** [review]
I approve as it seems okay even though I don't really understand everything. I'll let Hatim decide

File: webapp/.eslintrc.cjs [hunk 1]
```diff
@@ -12,10 +12,17 @@ module.exports = {
     "plugin:react/jsx-runtime",
     "plugin:react-hooks/recommended",
     "plugin:jsdoc/recommended-typescript",
-    "plugin:prettier/recommended",
+    "plugin:prettier/recommended", // Must be the last one
```
Comment (hdinia) [inline webapp/.eslintrc.cjs:15]: what you mean, in order it should be the last ?

**Review by hdinia — CHANGES_REQUESTED** [review]
could you also commit the `license-header.js` file ?

**Comment by MartinBelthle** [comment]
@hdinia I already merged the `license-header.js` file on dev in the last commit

File: webapp/.eslintrc.cjs (hunk 1 above)
Comment (skamril) [reply webapp/.eslintrc.cjs:15]: cf. https://github.com/prettier/eslint-plugin-prettier?tab=readme-ov-file#configuration-legacy-eslintrc

**Comment by hdinia** [comment]
> @hdinia I already merged the `license-header.js` file on dev in the last commit

ok nice then

**Review by hdinia — APPROVED** [review]

**Pull request merged**